portal/app/farm_updater/egauge_api.py

```diff
@@ -17,9 +17,10 @@ def __init__(self, url=None, username=None, password=None, t_sample=5):
         # Initializing parameters
         self.t_sample = t_sample
         self.keys_db = ['raw', 'processed']
-        self.keys = ['L1 Voltage', 'L2 Voltage', 'Power Circuit 1', 'Power Circuit 1*', 'Power Circuit 2',
-                     'Power Circuit 2*', 'Power Circuit 1 neutral', 'Shed Power', 'Control Fan Power',
-                     'Control Fan Power*', 'ts']
+        self.keys = ['L1 - VOLTAGE_C', 'L2 - VOLTAGE_A', 'POWER_CIRCUIT1', 'POWER_FACTOR_CIRCUIT1', 'POWER_CIRCUIT2',
+                     'POWER_FACTOR_CIRCUIT2', 'POWER_CIRCUIT1_NEUTRAL', 'SHED_POWER', 'CONTROL_FAN_POWER', 'ts', 'timestamp']
+
+        self.db_delete = False
```

Comment:
When is this ever not `False`?

Reply:
this is jut a flag to run the delete function when needed. It will have to be changed by hand. Need to think about a clever way to do this and figure out under which scenarios we would like to delete it. The only one I can think of is if we change the egauge naming conventions - which should not happen anymore.

Reply:
ok. So I'll remove this functionality and create a new branch to deal with the delete.